src/Microsoft.AspNetCore.Razor.Runtime/Runtime/TagHelpers/TagHelperExecutionContext.cs

```diff
@@ -183,18 +183,9 @@ public void AddMinimizedHtmlAttribute(string name)
                 throw new ArgumentNullException(nameof(name));
             }
 
-            HTMLAttributes.Add(
-                new TagHelperAttribute
-                {
-                    Name = name,
-                    Minimized = true
-                });
-            AllAttributes.Add(
-                new TagHelperAttribute
-                {
-                    Name = name,
-                    Minimized = true
-                });
+            var attribute = new TagHelperAttribute(name);
+            HTMLAttributes.Add(attribute);
```

Comment:
Naming of this doesn't make .Net guidelines for abbreviations btw. It's supposed to be HtmlAttributes